### ISSUE TITLE
Cloudfared?

### ISSUE THREAD

**Issue by rohanahmed69** [KasRoudra/Local2Internet#1]
When I started the tunnels, cloudfared was not there please solve this issue. Thank You.
![Screenshot_20211024-231402.png](https://user-images.githubusercontent.com/87943481/138607170-42edd706-0b1a-40c6-86ad-e8d81e9588e9.png)

**Comment by rohanahmed69** [comment]
> @rohanahmed69  Rerun the tool with `ruby l2in.rb`. If still same problem appears, use `uname -m` and `cd $HOME/.cffolder && ls`. Let me know the outputs!

Yes, I did your steps and found cloudflared in the given directory..
I saw that cloudflared was not in executable form. So I did **chmod +x cloudflared** and then it worked.
Thanks

**Comment by rohanahmed69** [comment]
> @rohanahmed69  Rerun the tool with `ruby l2in.rb`. If still same problem appears, use `uname -m` and `cd $HOME/.cffolder && ls`. Let me know the outputs!

And I wanted to ask you that can we use this for port forwarding? and how?

**Comment by KasRoudra** [comment]
@rohanahmed69 If it is working properly, why you need port forwarding? Cloudflared and ngrok are already using that.

**Comment by mokkabca** [comment]
![image](https://user-images.githubusercontent.com/95523018/186230978-a317fbfd-5c66-4ee2-baa1-9af4aff21b15.png)
 i am not able to use Cloudflared URL , i tried [above](https://github.com/KasRoudra/Local2Internet/issues/1#issuecomment-955248040) method

**Comment by KasRoudra** [comment]
@mokkabca Delete old one and clone again. I've updated the code

**Comment by mokkabca** [comment]
Thank you its working fine
